src/Symfony/Component/Dotenv/Tests/Command/DebugCommandTest.php

```diff
@@ -167,10 +167,27 @@ public function testWarningOnPhpEnvFile()
         $this->assertStringContainsString('[WARNING] Due to existing dump file (.env.local.php)', $output);
     }
 
-    private function executeCommand(string $projectDirectory, string $env): string
+    public function testLoadEnvFilesFromSubdirectory()
+    {
+        $projectDirectory = __DIR__.'/Fixtures/Scenario4';
+        file_put_contents($projectDirectory.'/composer.json', '{"extra":{"runtime":{"dotenv_path": "config/.env"}}}');
+
+        $output = $this->executeCommand($projectDirectory, 'dev', 'config/.env');
+
+        @unlink($projectDirectory.'/composer.json');
```

Comment:
Why is this necessary? Why don't we store the `composer.json` file in the fixtures directory in the repo?

Reply:
I did it before, but some tests failed for some reason. Please look at the previous commmit.